apps/web/public/static/locales/pt-BR/common.json

```diff
@@ -1940,4 +1940,25 @@
   "insights_subtitle": "Veja insights de reserva em seus eventos",
   "custom_plan": "Plano personalizado",
   "ADD_NEW_STRINGS_ABOVE_THIS_LINE_TO_PREVENT_MERGE_CONFLICTS": "↑↑↑↑↑↑↑↑↑↑↑↑↑ Adicione suas novas strings aqui em cima ↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑"
```

Comment:
Eh, how did this happen? - please uncommit the yarn.lock and move the fixes above this line :)